tests/test_cli.py

```
@@ -6,6 +6,7 @@
 
 # Import the package module (matches how users invoke it)
 import fetchtastic.cli as cli
+import fetchtastic.setup_config as setup_config
```

Ruff (0.15.12)

[warning] Use `from fetchtastic import setup_config` in lieu of alias

Replace with `from fetchtastic import setup_config`

(PLR0402)